homeassistant/components/webostv/__init__.py

```diff
@@ -191,12 +191,12 @@ async def async_update_options(hass, config_entry):
 
 async def async_control_connect(host: str, key: str | None) -> WebOsClient:
     """LG Connection."""
-    client = await WebOsClient.create(host, client_key=key)
+    client = WebOsClient(host, key)
     try:
         await client.connect()
-    except PyLGTVPairException as err:
+    except WebOsTvPairError as err:
         _LOGGER.warning("Connected to LG webOS TV %s but not paired", host)
-        raise PyLGTVPairException(err) from err
+        raise WebOsTvPairError(err) from err
```

Comment:
I think this is enough.
```suggestion
        raise
```